vendor/kubesphere.io/client-go

```diff
@@ -1 +0,0 @@
-../../staging/src/kubesphere.io/client-go
```

Comment:
The In-repo modules shouldn't be added to the vendor directory. it could be an issue of *pin-dependency.sh*. I will try to fix the script. Please move it out from the commit.

Reply:
Fixed